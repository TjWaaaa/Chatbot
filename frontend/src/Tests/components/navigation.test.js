// import { render, screen, cleanup } from '@testing-library/react';
// import renderer from 'react-test-renderer';
// import NavigationChatDetailMobile from '../../components/NavigationChatDetailMobile';
// import NavigationAllChatsMobile from '../../components/NavigationAllChatsMobile';
// import NavigationAllChatsWeb from '../../components/NavigationAllChatsWeb';

// afterEach(() => {
// 	cleanup();
// });

// test('Navigation test', () => {
// 	const mockedEvent = jest.fn();

// 	render(<NavigationChatDetailMobile name="Navigation Test" img="" event={mockedEvent} />);

// 	const element = screen.getByTestId('id1');
// 	expect(element).toBeInTheDocument();
// 	expect(element).toHaveTextContent('Navigation Test');
// });

// test('matches snapshot NavigationChatDetailMobile', () => {
// 	const mockedEvent = jest.fn();

// 	const tree = renderer
// 		.create(<NavigationChatDetailMobile name="Navigation Test" img="" event={mockedEvent} />)
// 		.toJSON();
// 	expect(tree).toMatchSnapshot();
// });

// test('matches snapshot NavigationAllChatsMobile', () => {
// 	const tree = renderer.create(<NavigationAllChatsMobile />).toJSON();
// 	expect(tree).toMatchSnapshot();
// });

test('matches snapshot NavigationAllChatsWeb', () => {
	expect(true).toBe(true);
});
