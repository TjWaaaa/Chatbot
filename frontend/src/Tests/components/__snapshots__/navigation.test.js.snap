// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot NavigationAllChatsMobile 1`] = `
<div
  className="pl-4 pr-4 pt-8 pb-4 bg-white sticky top-0 z-50"
  data-testid="id1"
>
  <div
    className="flex flex-row justify-between items-center"
  >
    <h1
      className="text-3xl font-bold"
    >
      All Chats
    </h1>
    <div>
      <div
        className="list-none"
      >
        <div
          className="h-11 w-11 overflow-hidden rounded-full cursor-pointer "
          onClick={[Function]}
        >
          <svg
            aria-hidden="true"
            className="flex h-12 w-12 pb-0.5 text-indigo-600"
            fill="currentColor"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clipRule="evenodd"
              d="M18.685 19.097A9.723 9.723 0 0021.75 12c0-5.385-4.365-9.75-9.75-9.75S2.25 6.615 2.25 12a9.723 9.723 0 003.065 7.097A9.716 9.716 0 0012 21.75a9.716 9.716 0 006.685-2.653zm-12.54-1.285A7.486 7.486 0 0112 15a7.486 7.486 0 015.855 2.812A8.224 8.224 0 0112 20.25a8.224 8.224 0 01-5.855-2.438zM15.75 9a3.75 3.75 0 11-7.5 0 3.75 3.75 0 017.5 0z"
              fillRule="evenodd"
            />
          </svg>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`matches snapshot NavigationAllChatsWeb 1`] = `
<div
  className="pl-4 pr-4 pb-4 bg-indigo-600 sticky top-0 z-50 h-16"
  data-testid="id1"
>
  <div
    className="flex flex-row justify-between items-center"
  >
    <h1
      className="text-3xl pt-2 font-semibold text-white"
    >
      All Chats are Bots
    </h1>
    <div
      className="pt-2.5"
    >
      <div>
        <div
          className="list-none"
        >
          <div
            className="h-11 w-11 overflow-hidden rounded-full cursor-pointer "
            onClick={[Function]}
          >
            <svg
              aria-hidden="true"
              className="flex h-12 w-12 pb-0.5 text-white"
              fill="currentColor"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clipRule="evenodd"
                d="M18.685 19.097A9.723 9.723 0 0021.75 12c0-5.385-4.365-9.75-9.75-9.75S2.25 6.615 2.25 12a9.723 9.723 0 003.065 7.097A9.716 9.716 0 0012 21.75a9.716 9.716 0 006.685-2.653zm-12.54-1.285A7.486 7.486 0 0112 15a7.486 7.486 0 015.855 2.812A8.224 8.224 0 0112 20.25a8.224 8.224 0 01-5.855-2.438zM15.75 9a3.75 3.75 0 11-7.5 0 3.75 3.75 0 017.5 0z"
                fillRule="evenodd"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`matches snapshot NavigationChatDetailMobile 1`] = `
<div
  className="pl-4 pr-4 pt-4 pb-4 bg-indigo-600 sticky top-0 z-50"
  data-testid="id1"
>
  <div
    className="flex flex-row justify-between items-center"
  >
    <div
      className="cursor-pointer"
      onClick={[Function]}
    >
      <svg
        aria-hidden="true"
        className="flex h-8 w-8 text-white"
        fill="currentColor"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clipRule="evenodd"
          d="M7.72 12.53a.75.75 0 010-1.06l7.5-7.5a.75.75 0 111.06 1.06L9.31 12l6.97 6.97a.75.75 0 11-1.06 1.06l-7.5-7.5z"
          fillRule="evenodd"
        />
      </svg>
    </div>
    <h1
      className="text-xl font-semibold text-white"
    >
      Navigation Test
    </h1>
    <img
      alt="Business Mann 42"
      className="h-12 w-12 rounded-full"
      src=""
    />
  </div>
</div>
`;
